package.json

```diff
@@ -35,10 +39,19 @@
     "html-webpack-plugin": "^3.2.0",
     "mini-css-extract-plugin": "^0.8.0",
     "node-sass": "^4.12.0",
+    "redux-promise-middleware": "^6.1.1",
     "sass-loader": "^7.1.0",
     "style-loader": "^0.23.1",
     "webpack": "^4.38.0",
     "webpack-cli": "^3.3.6",
     "webpack-dev-server": "^3.7.2"
+  },
+  "jest": {
+    "coveragePathIgnorePatterns": [
```

Comment:
@alainburindi why ignore these?

Reply:
```config/axios``` contains a line which can not be tested, it checks if the process is in development mode and send the ```axios``` instance accordingly, the same with ```src\Redux\store``` and the ```index``` files just store object

Reply:
I don't think they should be ignored @alainburindi.

Reply:
```
// frile configs/axios

import axios from 'axios';

let instance;
if (process.env.NODE_ENV !== 'test') {
  instance = axios.create({
    baseURL: 'https://authors-heaven.herokuapp.com/'
  });
}else{
  instance = axios;  
}
export default instance;
```

according to the above example the instruction in the if statement will never be reached since `NODE_ENV` equals `test`, which will decrease the coverage